chemprop/featurizers/atom.py

```diff
@@ -57,7 +57,7 @@ class MultiHotAtomFeaturizer(AtomFeaturizer):
     the final two) can increase in size depending on the input arguments.
     """
 
-    max_atomic_num: InitVar[int] = 100
+    atomic_nums: Sequence[int] = field(default_factory=lambda: list(range(1, 37)) + [53])
```

Comment:
I think this way of implementation will cause some tests to fail.

Reply:
Hi Oscar, thank you for looking at this. Can you point me to which tests you think this way of implementation will fail? It has worked in my personal tests. 

(Note that the automated tests fail because I didn't change `max_atomic_num` to `atomic_num` in `test_atom.py` like your [PR](https://github.com/chemprop/chemprop/pull/658) did. This wasn't intended to be a full PR, but I could change `test_atom.py` if you'd like to see if the automated tests pass.)